Update docs.md to prevent blank markup

frankfont opened the pull request:
The generated markup shows an empty string if you keep the curly braces --- reworded so the tag shows up and is still very clear to the reader.

frankfont committed:
Update docs.md to prevent blank markup

The generated markup shows an empty string if you keep the curly braces --- reworded so the tag shows up and is still very clear to the reader.

lcharette commented:
Hum... This create a new issue, as the `verbatim` tag in the code block underneath disappear now...

lcharette commented:
Ok, it's related to the metadata : https://github.com/userfrosting/learn/pull/55/files#diff-8c8072be8223454fb2b76f330f735efcL7

```
process:
    twig: true
never_cache_twig: true
```

Which is used to display the input mask demo... So we need a way to not process twig for those code block... :/

lcharette commented:
Ok. Found a [proper solution](https://twig.symfony.com/doc/2.x/templates.html#escaping). Pushing it up and merging this in `hotfix`. Thanks @frankfont for pointing this out !

lcharette committed:
Proper fix for the verbatim issue

Caused but the `process: twig: true` metadata. Need to escape the verbatim tags.
